packages/download-runtime/spec/lib/DownloadRuntimeSpec.js

```diff
@@ -0,0 +1,160 @@
+  let options;
+  let mockResponses;
+
+  beforeEach(() => {
+    downloadRuntime = new DownloadRuntime(fetchMock, cacheListProvider, runtimeVersionProvider);
+
+    options = {
+      dest: fse.mkdtempSync(path.join(os.tmpdir(), 'amp-download-')),
+    };
+
+    // Prepare Response objects for fetch mocks
+    mockResponses = {};
+    Object.keys(fakeFiles).forEach((filename) => {
+      const readable = new Readable();
+      readable.push(fakeFiles[filename]);
+      readable.push(null);
+
+      mockResponses[filename] = new crossFetch.Response(readable);
+    });
+  });
+
+  afterEach(() => {
+    fetchMock.reset();
+    if (fse.existsSync(options.dest) && fse.lstatSync(options.dest).isDirectory()) {
+      fse.removeSync(options.dest);
+    }
+  });
+
+  describe('getRuntime', () => {
+    it('downloads latest ampproject runtime by default', (done) => {
+      Object.keys(mockResponses).forEach((filename) => {
+        fetchMock.get(`${defaultHost}/${filename}`, mockResponses[filename]);
+        fetchMock.get(`${defaultHost}/rtv/${defaultRtv}/${filename}`, mockResponses[filename]);
+      });
+      downloadRuntime.getRuntime(options).then((ret) => {
+        expect(ret.status).toBe(true);
+        expect(ret.error).toBe('');
+        expect(ret.count).toBe(Object.keys(fakeFiles).length);
+        expect(ret.url).toBe(`${defaultHost}/rtv/${defaultRtv}/`);
+        expect(ret.dest).toBe(path.join(options.dest, 'rtv', defaultRtv));
+        expect(ret.rtv).toBe(defaultRtv);
+        for (let filename of Object.keys(fakeFiles)) {
+          filename = filename.split('/').join(path.sep);
+          const dest = path.join(options.dest, 'rtv', ret.rtv, filename);
+          expect(fse.existsSync(dest)).toBe(true);
+        }
+        done();
+      });
+    });
+
+    it('supports alternate hosts', (done) => {
+      const host = 'https://example.com/amp';
+      options['ampUrlPrefix'] = host;
+      Object.keys(mockResponses).forEach((filename) => {
+        fetchMock.get(`${host}/${filename}`, mockResponses[filename]);
+        fetchMock.get(`${host}/rtv/${defaultRtv}/${filename}`, mockResponses[filename]);
+      });
+      downloadRuntime.getRuntime(options).then((ret) => {
+        expect(ret.status).toBe(true);
+        expect(ret.url).toBe(`${host}/rtv/${defaultRtv}/`);
+        done();
+      });
+    });
+
+    it('supports manually specified runtime version', (done) => {
+      const rtv = '123';
+      options['rtv'] = rtv;
+      Object.keys(mockResponses).forEach((filename) => {
+        fetchMock.get(`${defaultHost}/${filename}`, mockResponses[filename]);
+        fetchMock.get(`${defaultHost}/rtv/${rtv}/${filename}`, mockResponses[filename]);
+      });
+      downloadRuntime.getRuntime(options).then((ret) => {
+        expect(ret.status).toBe(true);
+        expect(ret.url).toBe(`${defaultHost}/rtv/${rtv}/`);
+        done();
+      });
+    });
+
+    it('supports disabling destination dir clearing', (done) => {
```

Comment:
This test needs to be updated now that files are downloaded to `<dest>/rtv/<rtv>/` instead of `<dest>`.